### PR TITLE
autoscaled canary deployment blog

### PR DESCRIPTION
**Pull request by frankbu**
@geeknoid: please let me know what the rules/convensions are for the header fields that I haven't set: permalink, attribution, and redirect_from

### CODE REVIEW

File: _posts/2017-06-12-0.1-canary.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+---
+title: "Autoscaled Canary Deployment with Istio"
+overview: Canary deployment using Istio
+published: true
+permalink: ????
+attribution: ???? My name or "The Istio Team"
```
Comment (geeknoid) [inline _posts/2017-06-12-0.1-canary.md]: "The Istio Team"

File: _posts/2017-06-12-0.1-canary.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
+---
+title: "Autoscaled Canary Deployment with Istio"
+overview: Canary deployment using Istio
+published: true
+permalink: ????
```
Comment (rshriram) [inline _posts/2017-06-12-0.1-canary.md]: blog/canary-deployments-using-istio.html

File: _posts/2017-06-12-0.1-canary.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+attribution: ???? My name or "The Istio Team"
+layout: post
+type: markdown
+redirect_from: ????
```
Comment (rshriram) [inline _posts/2017-06-12-0.1-canary.md]: we don't need this

File: _posts/2017-06-12-0.1-canary.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
+---
+title: "Autoscaled Canary Deployment with Istio"
```
Comment (rshriram) [inline _posts/2017-06-12-0.1-canary.md]: Suggest "Canary Deployments using Istio"

**Pull request merged**